### PR TITLE
Add global human settlement layer entry

### PR DESCRIPTION
**Pull request by Kampen1988**

### CODE REVIEW

File: collections/global-human-settlement-layer-ghs-built-s2.yaml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    axis: y
+    extent:
+      - -56
+      - 72
+  t:
+    type: temporal
+    extent:
+      - '2018-01-01T00:00:00Z'
+      - '2018-12-31T00:00:00Z'
+    step: P5D
```
Comment (dthiex) [inline collections/global-human-settlement-layer-ghs-built-s2.yaml:83]: Would remove this line (it's not mandatory) as it means that there is a new image every 5 days in the specified extent (https://github.com/stac-extensions/datacube#temporal-dimension-object).

File: collections/global-human-settlement-layer-ghs-built-s2/README.MD [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+Built-up probability values rescaled in the range 0-100 (0 for probability = 0 and 100 for probability = 1); 255 = nodata
+
+## More information
+
+- [Official website](https://ghsl.jrc.ec.europa.eu/index.php)
+- [Data source](https://ghsl.jrc.ec.europa.eu/download.php?ds=buS2)
+- [Scientific publication on the dataset generation](https://link.springer.com/article/10.1007/s00521-020-05449-7)
+- [Global Human Settlement Layer - Understanding Human Presence on Planet Earth (booklet)](https://ghsl.jrc.ec.europa.eu/documents/GHSL_Brochure.pdf?t=1639151473)
+
+<img src="ghsl_paris.png" alt="GLobal Human Settlement Layer built-up probabilities over Paris" width="100%"/>
```
Comment (dthiex) [inline collections/global-human-settlement-layer-ghs-built-s2/README.MD:21]: Maybe add some empty lines after the image title so it's clearer to wish image the title belongs. Something like should work looking at how the html page is rendered [here](https://collections.sentinel-hub.com/global-human-settlement-layer-ghs-built-s2/readme.html).

```
<img src="ghsl_paris.png" alt="GLobal Human Settlement Layer built-up probabilities over Paris" width="100%"/>

*Global Human Settlement Layer built-up probabilities over Paris (France) visualised in EO Browser*
<br>

<img src="ghsl_overview.png" alt="GLobal Human Settlement Layer built-up probabilities over Germany, Netherlands and Belgium" width="100%"/>

*Global Human Settlement Layer built-up probabilities over parts of Germany, Netherlands and Belgium visualised in EO Browser*
```

File: collections/global-human-settlement-layer-ghs-built-s2.yaml [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+      -
+        - -180
+        - -56
+        - 180
+        - 72
+  temporal:
+    interval:
+      -
+        - '2018-01-01T00:00:00Z'
+        - '2018-12-31T00:00:00Z'
```
Comment (dthiex) [inline collections/global-human-settlement-layer-ghs-built-s2.yaml:64]: I saw we are not really consistent on this but I think the collection/datacube itself has not an extend of 1 year but exists only on 1 day (that 1 product of course is based on data from 1 year). I feel would be more correct here but am not 100 % sure.

```
       - '2018-01-01T00:00:00Z'
       - '2018-01-01T23:59:59Z
```

**Review by dthiex — CHANGES_REQUESTED** [review]

File: collections/global-human-settlement-layer-ghs-built-s2/README.MD (hunk 2 above)
Comment (Kampen1988) [reply collections/global-human-settlement-layer-ghs-built-s2/README.MD:21]: Done

**Review by dthiex — APPROVED** [review]

**Pull request merged**